src/AutoRest.CSharp/AutoRest/Plugins/Configuration.cs

```diff
@@ -34,10 +36,13 @@ public Configuration(string outputFolder, string? ns, string? name, string[] sha
         public bool ModelNamespace { get; }
         public bool HeadAsBoolean { get; }
         public bool SkipCSProjPackageReference { get; }
+        public string[] CredentialTypes { get; }
+        public string[]? CredentialScopes { get; }
         public static string ProjectRelativeDirectory = "../";
 
         public static Configuration GetConfiguration(IPluginCommunication autoRest)
         {
+            string[] defaultCredentialType = { "None"};
```

Comment:
Is "None" different than an empty array?

Reply:
Nope, changed it to empty array.